MPF.UI.Core/Windows/MainWindow.xaml.cs

```diff
@@ -313,6 +316,31 @@ public void ShowDebugDiscInfoWindow()
             return (result, submissionInfo!);
         }
 
+        /// <summary>
+        /// Show the Check Dump window
+        /// </summary>
+        public void ShowCheckDumpWindow()
+        {
+            // Hide MainWindow while Check GUI is open
+            this.Hide();
```

Comment:
This isn't done for anything else right now, is there a reason you chose to hide the main UI?

Reply:
The MPF.Check GUI seems to be independent of the main window, so felt it would be confusing to see both windows (e.g. thinking the system/media/drive on the main window might effect the Check Dump window).

Reply:
It's an interesting thought. I'll say that's good for now unless the user experience says otherwise 👍